src/main/java/seedu/address/logic/commands/AddPairCommand.java

```diff
@@ -1,21 +1,26 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NRIC_ELDERLY;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_NRIC_VOLUNTEER;
 
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.pair.Pair;
+import seedu.address.model.person.Elderly;
+import seedu.address.model.person.Volunteer;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.person.information.Nric;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to friendly link.
```

Comment:
```suggestion
 * Adds a person to FriendlyLink.
```
This is what we are calling the actual app if I'm not wrong. A small thing thing though.

Reply:
Resolved in 2099f8b1693a74ca9ac363377ff28b330d078ef5